Cli-CredentialHelper/Program.cs

```diff
@@ -105,6 +105,9 @@ internal static Version Version
         }
         private static Version _version;
 
+        // Storing OperationArguments here for the sake of Github ModalPrompts
+        private static OperationArguments operationArguments;
```

Comment:
I am not a fan of that myself either. The reason I did this is because I needed to check if the ModalUi config field is set to true. Another solution might be storing this data in the GithubAuthentication authority object and then passing both the modalui prompts and non-modalui prompts.